cfgov/scripts/http_smoke_test.py

```diff
@@ -240,6 +251,8 @@ def check_urls(base, url_list=None):
         logger.setLevel(logging.INFO)
     if args.base:
         BASE = args.base
+    if args.schema and args.host and args.port:
+        BASE = f"{args.schema}://{args.host}:{args.port}"
```

Comment:
this should probably be `BASE = f"{args.schema.lower()}://{args.host}:{args.port}"`